### PR TITLE
Luxianze/issue42

### LINKED ISSUE #42: Service Resolver

### PR DESCRIPTION
**Pull request by LUXIANZE**
resolves #42

### CODE REVIEW

File: src/resolvers/Service.js [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+      /**
+       * create if passes all validations
+       */
+      let newService = await Service.create(serviceInput);
+      newService = estimatedServiceTimeConverter(newService);
+
+      return newService;
+    },
+    updateService: async (root, { serviceInput }, context, info) => {
```
Comment (HowardYaw) [inline src/resolvers/Service.js:116]: validation on service name duplicated is not done here
suggest: create a const method for the service name validation check
something like roleValidator

File: src/resolvers/Service.js [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+      let service = await Service.findByIdAndUpdate(
+        serviceInput.id,
+        serviceInput,
+        { new: true }
+      );
+      service = estimatedServiceTimeConverter(service);
+
+      return service;
+    },
+    deleteService: async (root, { id }, context, info) => {
```
Comment (HowardYaw) [inline src/resolvers/Service.js:171]: delete option might not be available
if really need the should be Admin type user
Service is not service of a company/branch
as ref to the schema.jsonc
Service is the service our app will provide

File: src/resolvers/Service.js (hunk 1 above)
Comment (LUXIANZE) [reply src/resolvers/Service.js:116]: ok agree

File: src/resolvers/Service.js (hunk 2 above)
Comment (LUXIANZE) [reply src/resolvers/Service.js:171]: the role checking is at line 180

Comment (LUXIANZE) [reply src/resolvers/Service.js:171]: I should update the comment at line 162 to brand owner only

**Review by HowardYaw — APPROVED** [review]

**Pull request merged**